sdk/ai/Azure.AI.Projects.OneDP/src/Custom/Agent/AgentsClient.cs

```diff
@@ -1,178 +1,178 @@
-        {
-            // Create AgentConfigurationOptions
-            var agentConfigurationOptions = new AgentConfigurationOptions
-            {
-                AgentModel = new AzureAgentModel(modelId),
-                Instructions = instructions
-            };
-
-            // Create RunInputs
-            var inputMessages = new List<ChatMessage>
-            {
-                new UserMessage(new List<AIContent>
-                {
-                    new TextContent(message)
-                })
-            };
-            var runInputs = new RunInputs(inputMessages);
-
-            // Call the existing RunAsync method
-            return await RunAsync(agentConfigurationOptions, runInputs, cancellationToken).ConfigureAwait(false);
-        }
-
-        /// <summary>
-        /// Runs an agent with the specified agent ID and message.
-        /// </summary>
-        /// <param name="agentId">The ID of the agent to use for the run.</param>
-        /// <param name="message">The input message for the agent.</param>
-        /// <param name="cancellationToken">The cancellation token to use.</param>
-        /// <returns>The response containing the run result.</returns>
-        public virtual Response<Run> Run(string agentId, string message, CancellationToken cancellationToken = default)
-        {
-            // Create RunInputs
-            var inputMessages = new List<ChatMessage>
-            {
-                new UserMessage(new List<AIContent>
-                {
-                    new TextContent(message)
-                })
-            };
-            var runInputs = new RunInputs(agentId, inputMessages, null, null, null, null, null);
-
-            // Call the existing Run method
-            return Run(new AgentConfigurationOptions(), runInputs, cancellationToken);
-        }
-
-        /// <summary>
-        /// Asynchronously runs an agent with the specified agent ID and message.
-        /// </summary>
-        /// <param name="agentId">The ID of the agent to use for the run.</param>
-        /// <param name="message">The input message for the agent.</param>
-        /// <param name="cancellationToken">The cancellation token to use.</param>
-        /// <returns>The response containing the run result.</returns>
-        public virtual async Task<Response<Run>> RunAsync(string agentId, string message, CancellationToken cancellationToken = default)
-        {
-            // Create RunInputs
-            var inputMessages = new List<ChatMessage>
-            {
-                new UserMessage(new List<AIContent>
-                {
-                    new TextContent(message)
-                })
-            };
-            var runInputs = new RunInputs(agentId, inputMessages, null, null, null, null, null);
-
-            // Call the existing RunAsync method
-            return await RunAsync(new AgentConfigurationOptions(), runInputs, cancellationToken).ConfigureAwait(false);
-        }
-    }
-}
+//#nullable disable
```

Comment:
We shall remove Agents custom code from https://github.com/Azure/azure-sdk-for-net/tree/feature/azure-ai-projects-1dp, which means you can remove all Agents custom code